cdap-docs/developers-manual/source/building-blocks/datasets/index.rst

```diff
@@ -27,13 +27,17 @@ forcing you to manipulate data with low-level APIs, Datasets provide
 higher-level abstractions and generic, reusable implementations of
 common data patterns.
 
-The core Dataset of the CDAP is a Table. Unlike relational database
-systems, these tables are not organized into rows with a fixed schema.
-They are optimized for efficient storage of semi-structured data, data
-with unknown or variable schema, or sparse data.
-
-Other Datasets are built on top of Tables. A Dataset can implement
-specific semantics around a Table, such as a key/value Table or a
+The core Datasets of CDAP are Tables and FileSets:
+- Unlike relational database systems, CDAP tables are not organized into rows with a fixed schema.
```

Comment:
You could consider making "tables" and "file sets" in bold in the two bulleted 'graphs to highlight them in the text.